concepts/80 Troubleshooting/00 Basics/10 Check Requests in the Network Tab.md

```diff
@@ -0,0 +1,23 @@
+DevExtreme data-bound components may not display data or display unexpected data. The components use network requests to communicate with servers.
+
+### Issue 
+
+The DataGrid implements a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/). The **DataGrid.**[remoteOperations](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/remoteOperations/) property and [paging](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/paging/) are enabled. But the DataGrid loads and displays all pages at once.
+
+### Solution 
+
+Check request parameters and data that DataGrid receives from the server:
+
+1. Open your browser's DevTools and navigate to the Network tab.
+
+2. Reload the page. There will be multiple networks requests.
```

Comment:
In general, we try to avoid the following constructions at the beginning of a sentence: "There is", "There are", "There will be", etc. 
 
```suggestion
2. Reload the page. Review the request list.
```